### ISSUE TITLE
test(CL Fess): updateFeeAccumulatorPosition

### ISSUE THREAD

**Issue by p0mvn** [osmosis-labs/osmosis#3980]
## Background

We need to test `updateFeeAccumulatorPosition`

## Suggested Design

Assume a base position with pool id, owner, lower tick, and upper tick

Some test vectors that should be considered relative to base:
- updating a position with a different pool id implies updating a different position
- updating a position with a different owner implies updating a different position
- updating a position with a  different lowe tick implies updating a different position
- updating a position with an upper tick implies updating a different position

Basically, we want to test that prefixing is working correctly for positions. We have a related test here:
https://github.com/osmosis-labs/osmosis/blob/a3f80b7b14ad43ba528878833d9d151d15c2a19f/x/concentrated-liquidity/fees_test.go#L103-L120

Also, test that positions accumulator and liquidity are updated correctly.

## Acceptance Criteria
- tests added
- suggested test vectors covered
- improve godoc

**Comment by RusAkh** [comment]
hello @p0mvn is this free to pick up?

**Comment by p0mvn** [comment]
Hi @RusAkh . Thanks for checking in. @mattverse and I are working on fees and have the context. For the speed of iteration, it's best if one of us drives this to completion
